lab-stuart/README.md

```diff
@@ -0,0 +1,19 @@
+##  Queue - Lab Stuart
+
+## About
+This is a Queue data structure implementation.
+
+## Setup
+`const getQueue = require('../model/queue.js');`
+`let queue = getQueue();`
+
+## Methods:

```

Comment:
Add information about the return value for enqueue and what happens in edge cases. For example. What happens if you have enqueue(undefined);